internal/provider/resource_eva_role_permissions.go

```diff
@@ -0,0 +1,234 @@
+package provider
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-go/tftypes"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+
+	"github.com/mad-it/terraform-provider-eva/internal/eva"
+)
+
+type rolePermissionsType struct{}
+
+func (t rolePermissionsType) GetSchema(ctx context.Context) (tfsdk.Schema, diag.Diagnostics) {
+	return tfsdk.Schema{
+		MarkdownDescription: "Eva role permissions configuration.",
+
+		Attributes: map[string]tfsdk.Attribute{
+			"id": {
```

Comment:
To be able to run acceptance tests every resource must have a `id` attributes (even when it's not being used), `role permissions` doesn't have a unique ID so for now I'm setting `RoleID` into id attribute.

https://www.terraform.io/plugin/framework/acctests#implement-id-attribute